### DISCUSSION TITLE
Hotspots in 360 photo

### DISCUSSION THREAD

**Discussion by inst5** [cables-gl/cables_docs#141] [Answers & Questions] [answered]
Hi,
I am trying to do kind of  a 360 virtual tour with a 360 photo.
So the navigation works wells. But can I also stick hotspots link to my photo?
For this I would need to provide the xy-coordinates for the hotspot. 

At the end I would like to do sth. similar as Marzipano 360 views (https://www.marzipano.net/) does.

thanks for any help!

**Comment by pandrr** [comment, marked as answer]
yes, its quite easy

you could do something like this:

https://cables.gl/edit/p0sz2Y


hf

**Comment by inst5** [comment]
Hi,
oh great. - I did got it, how did this.
But after watching the Cast ray tutorial : https://www.youtube.com/watch?v=dF1fP_7fh4Q
I understand how to solve this.
thanks!
